en-US/Free-Range_Monsters/themes/pdf-theme.yml

```diff
@@ -0,0 +1,46 @@
+extends:
+  - default
+base:
+  font-size: 12
+page:
+  size: Letter
+title-page:
+  logo:
+    align: left
+    image: image:../../../assets/Free5e-Logo-Gold-Ring.png[pdfwidth=20%]
+    top: 0%
+  title:
+    font-color: #b77a2b
+    font-family: Helvetica
+    font-size: 32
+    font-style: bold
+    top: 15%
+  authors:
+    content:
+      name_only: '{author}'
+    font-color: #29326a
+    font-family: Helvetica
+    font-size: 20
+footer:
```

Comment:
Here's a page with a footer using the current settings:
<img width="683" height="888" alt="image" src="https://github.com/user-attachments/assets/e765b78f-57f9-4b65-8d66-2655d6f5a5dc" />